src/styles/scss/internal.scss

```diff
@@ -120,3 +120,11 @@ code {
 .internal-arrow-box:hover {
   background-color: #f5f8fa;
 }
+
+.sb-main-padded {
+  padding: 0 !important;
+}
+
+.sb-show-main {
+  padding: 0 !important;
+}
```

Comment:
`!important` should not be used lightly. Could you add a comment explaining why it is necessary?

Reply:
Storybook related classes to setup frame / board